internal/distro/rhel8/azure.go

```diff
@@ -571,3 +576,33 @@ var azureRhuiImgType = imageType{
 	exports:             []string{"archive"},
 	basePartitionTables: azureRhuiBasePartitionTables,
 }
+
+func SapAzureImageConfig(rd distribution) *distro.ImageConfig {
+	return SapImageConfigX86_64(rd).InheritFrom(defaultAzureImageConfig)
+}
+
+// TODO What to put in here
+func azureSapImgType(rd distribution) imageType {
+	return imageType{
+		name:     "azure-sap",
+		filename: "disk.vhd.xz",
+		mimeType: "application/xz",
+		packageSets: map[string]packageSetFunc{
+			// the ec2 buildroot is required due to the cloud-init stage and dependency on YAML
+			buildPkgsKey: ec2BuildPackageSet,
+			osPkgsKey:    azureSapPackageSet,
+		},
+		packageSetChains: map[string][]string{
+			osPkgsKey: {osPkgsKey, blueprintPkgsKey},
+		},
+		defaultImageConfig:  SapAzureImageConfig(rd),
+		kernelOptions:       defaultAzureKernelOptions,
+		bootable:            true,
+		defaultSize:         4 * common.GibiByte,
+		pipelines:           vhdPipelines(false),
+		buildPipelines:      []string{"build"},
+		payloadPipelines:    []string{"os", "image", "vpc"},
+		exports:             []string{"vpc"},
```

Comment:
There is missing the `archive` pipeline, which should be also exported if the image is supposed to be produced as an XZ archive.